ui/tests/data-collection.spec.ts

```diff
@@ -5,7 +5,7 @@ import { expect, test } from './fixtures';
 
 test.describe('Dataset', () => {
     test('list items', async ({ page }) => {
-        await page.goto('/dataset');
+        await page.goto('project/some-id/dataset');
```

Comment:
Assuming you'd go with `/projects/some-id/dataset` then you'd need to update these